app/features/history/history_filter/dateRange/endDate/EditHistoryFilterEndDateActions.js

```diff
@@ -0,0 +1,15 @@
+import {
+    SAVE_HISTORY_FILTER_END_DATE,
+    DISMISS_HISTORY_FILTER_END_DATE,
+} from 'app/configs+constants/ActionTypes';
+
+export const changeDate = (date) => ({ 
+    type: SAVE_HISTORY_FILTER_END_DATE,
+    date: date.toISOString().split('T')[0],
```

Comment:
- [ ] this would result in the wrong timezone. toLocaleString should get you the right timezone in theory, needs testing, though you'd have to format it for display purposes

Reply:
React-Native has poor support for toLocaleString especially formatting wise, not sure why. toString() works in preserving the users timezone so going with that.